### DISCUSSION TITLE
[Bug] ERROR Cannot bind argument to parameter 'LinkName' because it is an empty string.

### DISCUSSION THREAD

**Discussion by littleli** [ScoopInstaller/Scoop#5408] [Help and Questions]
## Bug Report

I'm maintaining successful bucket https://github.com/littleli/scoop-clojure.

Recently one of my users reported following issue: https://github.com/littleli/scoop-clojure/issues/171

I run scoop on two of my boxes and I haven't met anything like that. So I'm asking for council. My suspicion is that there is unescaped file-path somewhere in the process, but there is nothing obvious from the user's report. Further user info may be required, but allow me to post this anyway. Any suggestion welcome.

**Comment by slickag** [comment]
I ran into the same problem when trying to update the posh-git PS module. I had to manually remove the junction in the ~\scoop\modules directory that was pointing to a non-existing ~\scoop\apps\module-name\current folder to resolve the issue.
